.gitignore

```diff
@@ -258,4 +258,5 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+VDF.GUI/Properties/launchSettings.json
```

Comment:
This is an unwanted change

Reply:
Removed.
That was my bad attempt to not include changes to launchSettings.json in this update.

But I have a question on this issue.  I could not get Visual Studio 2022 to start VDF in debug mode without adding the working directory in the launchSettings.json file. How are you able to start the program in Visual Studio without specifying a working directory?

Reply:
reversed change.